HighSchoolDB/3 Reports.sql

```diff
@@ -0,0 +1,32 @@
+
+use HighSchoolDB
+go
+
+-- 1. Show the number of students per zip code
+SELECT ZipCode, COUNT(*) AS NumberOfStudents
+FROM Students
+GROUP BY ZipCode;
+
+-- 2. Show a list of students who have already sent in the full tuition ($15,000)
+SELECT FirstName, LastName, Grade, Class, TuitionReceived
+FROM Students
+WHERE TuitionReceived = 15000;
+
+-- 3. Show a list of girls who have at least one sister in school with them
+select s.LastName as SistersInSchool
+from Students s
+group by s.LastName
```

Comment:
Some students have the same last name but different addresses. They may be cousins but  they are not sisters

Reply:
Updated

Reply:
look great! thanks